### ISSUE TITLE
[Quest] [Warlock] Knowledge of the Orb of Orahil

### ISSUE THREAD

**Issue by heaventhegod17** [dalaranwow/dalaran-wow#5684]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
So basically i was doing the Knowledge of the Orb of Orahil, i went to the Barrens, chose the Infernal Path, went all the way to Desolace and captured the infernal as required, then i was sent all the way to Dustwallow Marsh where i had to kill the Felguard in order to finish the mission, i failed the quest by not being able to kill the Felguard. I abandoned the quest to try and do it again but i am now at the quest giver and i no longer see the quest.
**Expected behaviour**: Tell us what should happen instead
I should be able to do the quest again.

**Steps to reproduce the problem**:

1.  Fail the quest to kill the felguard.
2. Go to Ironforge at the quest giver.

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/47227805/52121778-7631ca80-2629-11e9-86a4-427029f1a95d.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Visit the correct quest giver and the NPC will offer it to you again. 
Closing
